pkg/commands/workload_test.go

```diff
@@ -1867,7 +1869,7 @@ Published source
 			opts.LoadDefaults(c)
 			opts.DefineFlags(ctx, c, cmd)
 			cmd.ParseFlags(test.args)
-
+			ctx = logger.StashProgressBarLogger(ctx, testprogresslogger)
```

Comment:
any reason why its not loaded into context right after `testProgressLogger` is declared

Reply:
I wanted to reuse the same Progress bar logger instance for all test rather than redefining for each tests